index.js

```diff
@@ -107,6 +107,13 @@ export const Link = (props) => {
         event.button !== 0
       )
         return;
+      
+      // ignores the navigation when routing is outside
+      // of application
+      const target = new Url(event.target.href, window.location.origin);
```

Comment:
That second argument is ignored if the URL contains the host

<img width="666" alt="Screen Shot 2020-12-26 at 13 01 03" src="https://user-images.githubusercontent.com/10779424/103151928-7b96e180-477a-11eb-8e54-cc2ec16c9d39.png">